edge/pkg/edged/edged.go

```diff
@@ -121,12 +121,20 @@ func (e *edged) Enable() bool {
 func (e *edged) Start() {
 	klog.Info("Starting edged...")
 
-	// edged saves the data of mqtt container in sqlite3 and starts it. This is a temporary workaround and will be modified in v1.15.
-	withMqtt, err := strconv.ParseBool(os.Getenv(constants.DeployMqttContainerEnv))
-	if err == nil && withMqtt {
-		err := dao.SaveMQTTMeta(e.nodeName)
-		if err != nil {
-			klog.ErrorS(err, "Start mqtt container failed")
+	// edged saves the data of mqtt container in sqlite3 and starts it.
```

Comment:
The code nesting here is a little deep and simply serves as interim logic. I think it can be extracted into a function for later replacement. It also improves the readability of the code. : )

Reply:
> The code nesting here is a little deep and simply serves as interim logic. I think it can be extracted into a function for later replacement. It also improves the readability of the code. : ) P.S. The env judgment can be extracted as a function and added to the original if condition.

Thanks, fixed in #5101